core/auth/index.ts

```diff
@@ -144,6 +144,16 @@ async function authorize(credentials: unknown): Promise<User | null> {
   }
 }
 
+const partitionedCookie = (name?: string) =>
+  ({
+    ...(name !== undefined ? { name } : {}),
```

Comment:
I assume this will be used on other PR?

Reply:
@fikrikarim Great question, no, this is just to make life easier for whomever inherits this code from us.